python/pyspark/sql/session.py

```diff
@@ -76,6 +76,7 @@
     from pyspark.sql.udf import UDFRegistration
     from pyspark.sql.udtf import UDTFRegistration
     from pyspark.sql.datasource import DataSourceRegistration
+    from pyspark.sql.profiler import Profile
```

Comment:
Why is this here? Move to line 50?

Reply:
Adjusted. I noticed we intended to import inside a `property` as well as under TYPE_CHECKING, like `dataSource` and `udtf`, `udf`, etc. But now I believe that is not a convention.